### DISCUSSION TITLE
New version available, 28 March 2023

### DISCUSSION THREAD

**Discussion by biglimp** [UMEP-dev/UMEP#511] [Announcement]
New features in this version (4.0 for Menu-based and 2.0 for Processing):
- New semi-empirical wind model included (URock)
- New version of SOLWEIG (2022a)
- New tool to estimate spatial variations of thermal comfort (SpatialTC)
- As from this version, many of the tools has been moved over to UMEP for Processing to reduce inconsistencies
- New tutorials available (https://umep-docs.readthedocs.io/projects/tutorial/en/latest/)

Upcoming features! New version of SUEWS along with SUEWS database plugin and pre-processing tools.

**Comment by AlexKDo** [comment]
Hi! I am trying to install the new version from here (zip) and i get an error.
![image](https://user-images.githubusercontent.com/81244161/230403977-f187ef86-6160-4d30-bb54-e54c55221b72.png)

**Reply by biglimp** [reply]
You need to display the actual error. Now you show the General tab.

**Comment by AlexKDo** [comment]
![image](https://user-images.githubusercontent.com/81244161/230489042-f4628349-e497-4abe-bf2b-eba3a8eb76a0.png)
![image](https://user-images.githubusercontent.com/81244161/230489097-da5268eb-1c58-409f-9e8d-bc529680b191.png)

**Comment by j3r3m1** [comment]
You need to install a required library called "jaydebeapi". To achieve this you need to do a "pip install jaydebeapi" in the osgeo command shell. You will probably have to do that with the "numba" and "netcdf4" libraries.

**Comment by biglimp** [comment]
We just released a fix (version 2.0.1) where UMEP automatically installs dependent python modules. Should now be available from the QGIS plugin repository (Manage and Install Plugins).

**Reply by AlexKDo** [reply]
i reinstalled version 2.0.1 but error 

![image](https://user-images.githubusercontent.com/81244161/230763770-33ea3830-5276-4f33-b135-c0f05adaa4a6.png)
![image](https://user-images.githubusercontent.com/81244161/230763800-2e2d27c1-c067-4526-a569-ed93811a89f9.png)

**Comment by AlexKDo** [comment]
I fixed the problem install all the missing libraries manually from osgeo command shell and updated Java from 32 to 64 bit. Thank you very much I am very happy to work with the new updates!

**Comment by Luigi-MEX** [comment]
Can anyone explain better how to install the plugin UMEP for Processing in QGIS? I'm trying to install it, but it doesn't work. 

Thanks in advance!

**Reply by biglimp** [reply]
We need more information to help you. What Operating system do you use? What version of QGIS do you have? What version of UMEP are you trying to install? What is the error message?

**Reply by Luigi-MEX** [reply]
Ok, I solved the problem by installing the last version of the UMEP model: Version: [2079] UMEP for processing 2.0.3. However, no icon or name is displayed on the horizontal bar.

**Reply by biglimp** [reply]
Horizontal bar? You mean in the Processing panel in QGIS. Go to Manage and Install Plugins and see if UMEP for Processing is ticked in.

**Reply by Luigi-MEX** [reply]
Yes, it's ticked.

**Reply by biglimp** [reply]
Tick it off and on again and see if any error message appear. Also try to restart QGIS

**Reply by Luigi-MEX** [reply]
OK, thanks! I did find it in the processing panel!
